sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java

```diff
@@ -106,6 +124,16 @@ public CaseImportExportSteps(
           webDriverHelpers.clickOnWebElementBySelector(NEW_EXPORT_CONFIGURATION_SAVE_BUTTON);
         });
 
+    And(
+        "I add ([^\"]*) data to export in existing Export Configuration for Custom Case Export",
```

Comment:
let' replace add with select, since the method since the method is selecting something from UI